app/db/crud/document.py

```diff
@@ -102,7 +102,9 @@ def _get_language_for_phys_doc(db: Session, physical_document_id: str) -> str:
         .filter(Language.id == PhysicalDocumentLanguage.language_id)
     ).one_or_none()
 
-    return cast(str, language.code) if language else ""
+    # I do not fully understand why the strip() below is required, but without it
+    # we see trailing spaces on the language code.
```

Comment:
Can anyone explain this?!

Reply:
Language code is length 3? 
So if it's en it's entered as 'en ' ?

Reply:
good spot! should probably use the real value of "eng" in the test